app/main.py

```diff
@@ -1,2 +1,59 @@
+from typing import Hashable, Any
+
+
 class Dictionary:
-    pass
+    def __init__(self, capacity: int = 8) -> None:
+        self.capacity = capacity
+        self._size = 0
+        self.table = [[] for _ in range(self.capacity)]
+
+    def get_hash(self, key: Any) -> int:
+        return hash(key) % self.capacity
+
+    def __setitem__(self, key: Any, value: Hashable) -> None:
```

Comment:
```suggestion
    def __setitem__(self, key: Hashable, value: Any) -> None:
```

Reply:
In dictionary key must indeed be hashable, but there are no specific restrictions on the hashability of the values. Values in a dictionary can be of any type, whether hashable or not